Makefile

```diff
@@ -1,12 +1,72 @@
+#!/usr/bin/make
+TAILWIND_ARGS := -i styles/tailwind.css -c styles/tailwind.config.js -o _site/css/style.css
+VIRTUALENV := venv/bin/activate
```

Comment:
Is the `venv` addition really important to you? Do you personally find it helpful?

I use `conda`, and I am worried this might cause problems with whatever people's alternate virtual environments setups are. But, if it is important to you or to the code on Netlify, I am okay with keeping it.

Reply:
I think it's important to have a virtual environment created by default for reproducibility, but we can make it optional. It's currently called by default on `make install`.

`make pip` will install the python dependencies without creating a virtual environment.

Regarding Netlify, I had to add if wrappers to check for `./venv` because Netlify's build image doesn't include `venv`. This way it also works if you haven't set up `./venv`.

Reply:
> but we can make it optional

I would really love if it was possible for it to be optional! But, I agree with your point that virtual environments are helpful.

Reply:
> Regarding Netlify, I had to add if wrappers to check for ./venv because Netlify's build image doesn't include venv. This way it also works if you haven't set up ./venv.

Cool =)

Reply:
I went ahead and made `venv` optional, and updated the readme to make the option clear.